components/products/Product.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { Product } from '@prisma/client';
+
+defineProps<{
+  product: Product;
+}>()
+
+</script>
+
+<template>
+  <div class="product">
+
+    <img v-if="product.image" :src="product.image" alt="Product Image" class="image" />
+    <div v-else class="image-placeholder"></div>
```

Comment:
Right, in this case, there would be "alt" text displayed. To prevent this, I would use @error tag and clean up the code, somewhat like this:

const imagePlaceholder = 'https://via.placeholder.com/600x400?text=Product+Image+Not+Found'

const handleImageError = (err: Event) => {
  (err.target as HTMLImageElement).src = imagePlaceholder;
}

<img :src="product.image ?? imagePlaceholder" alt="Product Image" class="image" @error="handleImageError"/>

Reply:
I was thinking about maybe conditionally putting `image-background` for the `<img />` or something like that  - in general, i would try to go with the most css-like way.